src/alfasim_sdk/testing/fixtures.py

```diff
@@ -59,14 +59,21 @@ def check_base_case_is_loaded(self) -> None:
     def check_base_case_is_not_loaded(self) -> None:
         assert self._case is None, "Base case already loaded"
 
-    def load_base(self, *parts: Union[str, Path]) -> None:
+    def load_base(
+        self,
+        *parts: Union[str, Path],
+        case_description: Optional[CaseDescription] = None,
+    ) -> None:
         """
-        Define the base case from an alfacase file.
+        Define the base case from an alfacase file or a CaseDescription.
         """
         self.check_not_run()
         self.check_base_case_is_not_loaded()
 
-        self._case = convert_alfacase_to_description(Path(*parts))
+        if case_description is not None:
+            self._case = case_description
+        else:
+            self._case = convert_alfacase_to_description(Path(*parts))
```

Comment:
Properly typing this and preventing it to be wrongly used can be tricky.
Maybe we should be more explicit.

```python
    def check_can_load_base(self) -> None:
        self.check_not_run()
        self.check_base_case_is_not_loaded()

    def load_base_from_alfacase(self, path: Union[str, Path]) -> None:
        self.check_can_load_base()
        self._case = convert_alfacase_to_description(Path(path))
    
    def load_base_from_case_description(self, case_description: CaseDescription) -> None:
        self.check_can_load_base()
        self._case = case_description
```

(docstring omitted for the benefit of a short example)

Reply:
I think is a better idea. I will change it.

- [ ] Create explicit `load_case` function fro `alfacase` and `CaseDescription`